tests/integration/helpers/session-color-test.js

```diff
@@ -0,0 +1,26 @@
+import { module, test } from 'qunit';
+import { setupIntegrationTest } from 'open-event-frontend/tests/helpers/setup-integration-test';
+import hbs from 'htmlbars-inline-precompile';
+import { render } from '@ember/test-helpers';
+
+module('Integration | Helper | session-color', function(hooks) {
+  setupIntegrationTest(hooks);
+
+  test('it renders', async function(assert) {
+    this.set('inputState', 'confirmed');
+    await render(hbs`{{session-color inputState}}`);
+    assert.equal(this.element.textContent.trim(), 'green');
+
+    this.set('inputState', 'accepted');
+    await render(hbs`{{session-color inputState}}`);
+    assert.equal(this.element.textContent.trim(), 'yellow');
+
+    this.set('inputState', 'pending');
+    await render(hbs`{{session-color inputState}}`);
+    assert.equal(this.element.textContent.trim(), 'yellow');
+
+    this.set('inputState', '');
```

Comment:
make this 
`this.set('inputState' , 'rejected');`

Reply:
Did it